src/igz_controls/services/validation.service.js

```diff
@@ -181,16 +178,32 @@
                     name: 'noConsecutiveCharacters',
                     label: $i18next.t('common:NO_CONSECUTIVE_CHARACTER', {lng: lng}) + ': ' + convertToLabel(chars),
                     pattern: new RegExp('^' + convertedPattern)
-                }
+                };
             },
-            onlyAtTheBeginning: function (chars) {
-                var convertedPattern = convertToPattern(chars);
+            mustNotBeOneOf: function (words) {
+                var convertedPattern = words.split(' ').join('|');
 
                 return {
-                    name: 'onlyAtTheBeginning',
-                    label: $i18next.t('common:ONLY_AT_THE_BEGINNING', {lng: lng}) + ': ' + convertToLabel(chars),
-                    pattern: new RegExp('^([' + convertedPattern + '])?[^' + convertedPattern + ']+$')
+                    name: 'mustNotBeOneOf',
+                    label: $i18next.t('common:MUST_NOT_BE_ONE_OF', {lng: lng}) + ': ' + convertToLabel(words),
+                    pattern: new RegExp('^(?!' + convertedPattern + ').*|.*(?<!' + convertedPattern + ')$|^(' +
+                        convertedPattern + ').+$')
```

Comment:
Oh, wow, I actually didn't mean for a list of words separated by spaces in a single string literal.
I was aiming at an array of strings.
So it's easier for us to pass it into the `mustNotBeOneOf` method.
And we can use a function as the value of `pattern` instead of a `RegExp`.
Here's what I had in mind:

```js
mustNotBeOneOf: function (words) {
    return {
        name: 'mustNotBeOneOf',
        label: ...,
        pattern: function (value) {
            label: $i18next.t('common:MUST_NOT_BE_ONE_OF', { lng: lng }) + ': ' + words.join(', '),
            return lodash.includes(words, value);
        }
    };
}
```